objdetect.cpp

```diff
@@ -175,4 +175,140 @@ bool QRCodeDetector_DetectAndDecodeMulti(QRCodeDetector qr, Mat input, CStrings*
   decoded->length = decodedCodes.size();
   decoded->strs = strs;
   return res;
+}
+
+FaceDetectorYN FaceDetectorYN_Create(const char* model, const char* config, Size size) {
+    cv::String smodel = cv::String(model);
+    cv::String sconfig = cv::String(config);
+    cv::Size   ssize = cv::Size(size.width, size.height);
+
+    return new cv::Ptr<cv::FaceDetectorYN>(cv::FaceDetectorYN::create(smodel, sconfig, ssize));
+}
+
+FaceDetectorYN FaceDetectorYN_Create_WithParams(const char* model, const char* config, Size size, float score_threshold, float nms_threshold, int top_k, int backend_id, int target_id) {
+    cv::String smodel = cv::String(model);
+    cv::String sconfig = cv::String(config);
+    cv::Size   ssize = cv::Size(size.width, size.height);
+
+    return new cv::Ptr<cv::FaceDetectorYN>(cv::FaceDetectorYN::create(smodel, sconfig, ssize, score_threshold, nms_threshold, top_k, backend_id, target_id));
+}
+
+FaceDetectorYN FaceDetectorYN_Create_FromBytes(const char* framework, void* bufferModel, int model_size, void* bufferConfig, int config_size, Size size) {
+    cv::String sframework = cv::String(framework);
+    cv::Size   ssize = cv::Size(size.width, size.height);
+    
+    std::vector<uchar> bufferModelV;
+    std::vector<uchar> bufferConfigV;
+
+    uchar* bmv = (uchar*)bufferModel;
+    uchar* bcv = (uchar*)bufferConfig;
+
+
+    for(int i = 0; i < model_size; i ++) {
+        bufferModelV.push_back(bmv[i]);
+    }
+   for(int i = 0; i < config_size; i ++) {
+        bufferConfigV.push_back(bcv[i]);
+    }
+
+    return new cv::Ptr<cv::FaceDetectorYN>(cv::FaceDetectorYN::create(sframework, bufferModelV, bufferConfigV, ssize));
+}
+
+FaceDetectorYN FaceDetectorYN_Create_FromBytes_WithParams(const char* framework, void* bufferModel, int model_size, void* bufferConfig, int config_size, Size size, float score_threshold, float nms_threshold, int top_k, int backend_id, int target_id) {
+    cv::String sframework = cv::String(framework);
+    cv::Size   ssize = cv::Size(size.width, size.height);
+    
+    std::vector<uchar> bufferModelV;
+    std::vector<uchar> bufferConfigV;
+
+    uchar* bmv = (uchar*)bufferModel;
+    uchar* bcv = (uchar*)bufferConfig;
+
+
+    for(int i = 0; i < model_size; i ++) {
+        bufferModelV.push_back(bmv[i]);
+    }
+   for(int i = 0; i < config_size; i ++) {
```

Comment:
```suggestion
    for(int i = 0; i < config_size; i ++) {
```

Minor formatting nit.

Reply:
:man_facepalming: 
sorry.